### ISSUE TITLE
ETQ Usager je veux une notification (bulle orange) lorsque je reçois un message sur la messagerie

### ISSUE THREAD

**Issue by Benjamin-Doberset** [betagouv/demarches-simplifiees.fr#3952]
Situation actuelle : lorsqu'un message est déposé sur le fil de messagerie je reçois un mail mais il n'y a pas de bulle orange sur mon dossier et l'onglet messagerie
Situation souhaitée : une bulle orange apparaît sur mon dossier et sur l'onglet messagerie

**Comment by kemenaran** [comment]
Effectivement, on n'a jamais implémenté les pastilles de notification sur la partie usager.
